source/addComment.js

```diff
@@ -1,3 +1,33 @@
 $(document).ready(function(){
-  // add your code here
-})
+  $('#new_comment_button').on("click", function(event) {
+    event.preventDefault()
+    $('#comments').append(showCommentForm());
+  });
+
+  $("#new_comment").on("submit", function(event) {
+    event.preventDefault();
+    publishComment();
+  });
+});
+
+function showCommentForm() {
+  let formBegin = '<form name="new_comment" id="new_comment">';
+  let commentBody = '<textarea name="comment" placeholder="Tell us what you think..."></textarea>';
+  let author = '<text name="author" placeholder="Written by..."></text>';
+  let submit = '<input type="submit" value="Create Comment"/></form>';
+  return formBegin + commentBody + author + submit;
+};
+
+function publishComment() {
+  let comment = $('[name=comment]').val();
```

Comment:
you could even extract out these two lines clearing the values of the comment and author into its own function 👍